api-client/src/models/PoiModel.ts

```diff
@@ -118,7 +118,6 @@ class PoiModel extends ExtendedPageModel {
     return (
       this._category ??
       new PoiCategoryModel({
-        // eslint-disable-next-line no-magic-numbers
```

Comment:
I'm actually a little bit perplex why no-magic-numbers does not trigger here o.o

Reply:
Maybe `id` is some kind of ignore pattern now?

Reply:
https://eslint.org/docs/latest/rules/no-magic-numbers#detectobjects

That looks like it could be the reason? Its not active as default and therefore doesn't check objects if I understand correctly. Not sure why it triggered an error before then.

Reply:
Ahh. [This](https://eslint.org/docs/latest/rules/no-magic-numbers#detectobjects) needs to be set to true.

Reply:
I guess the option was added at some point and we didn't notice. I'll enable it again. 👍🏼